message/message.yml

```diff
@@ -0,0 +1,412 @@
+        default:
+          $ref: '#/responses/Error'
+      parameters:
+        - in: query
+          name: timeoutMilliseconds
+          description: The time to keep the connection open while waiting for a message
+          type: integer
+          required: false
+    delete:
+      tags:
+        - Message Queue
+      summary: Flush the message queue
+      description: Flushes the queue of received messages, discarding all messages without reading them.
+      operationId: DeleteMessages
+      responses:
+        200:
+          description: Success
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v1/sessions/{token}/configuration/queueSizeBytes:
+    parameters:
+      - $ref: '#/parameters/Token'
+    get:
+      tags:
+        - Message Queue
+      summary: Get the current queue size
+      description: Gets the current queue size (in bytes). This is helpful for determining the amount of memory that unread messages are consuming.
+      operationId: GetCurrentQueueSIze
+      responses:
+        200:
+          schema:
+            $ref: '#/definitions/QueueSizeBytes'
+          description: Success
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v1/sessions/{token}/configuration/maxQueueSizeBytes:
+    parameters:
+      - $ref: '#/parameters/Token'
+    get:
+      tags:
+        - Message Queue
+      summary: Get the maximum queue size
+      description: Gets the maximum queue size (in bytes). This is helpful for determining how much total memory is available for storing messages.
+      operationId: GetMaximumQueueSize
+      responses:
+        200:
+          schema:
+            $ref: '#/definitions/MaxQueueSizeBytes'
+          description: Success
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v1/websocket:
+    get:
+      tags:
+        - WebSocket (SystemLink Server only)
+      summary: Open a WebSocket session
+      description: Opens a persistent connection to the web server that allows bi-directional communication using a JSON
+                   formatted protocol. Once the connection is open, available actions include the WebSocket Publish Action,
+                   WebSocket Subscribe Action, and WebSocket Unsubscribe Action. The schema for these actions is defined in
+                   the *Models* section. Note that WebSocket functionality is currently only supported in SystemLink Server.
+      operationId: OpenWebSocketSession
+      responses:
+        101:
+          description: Switching Protocols
+          headers:
+            Sec-WebSocket-Accept:
+              description: Used in the WebSocket opening handshake. Set to a value derived from the Sec-WebSocket-Key header.
+              type: string
+        426:
+          description: Upgrade Required
+        default:
```

Comment:
A websocket route will never return a JSON error response. If the web service rejects the connection for any reason, the client will receive a 403 (though I don't think we have any specific failure reasons, so probably don't need to document it explicitly?).